samples/dashboards/demo/cloud-monitoring/demo.js

```diff
@@ -0,0 +1,724 @@
+                                width: '100%'
+                            },
+                            medium: {
+                                width: '100%'
+                            },
+                            large: {
+                                width: '50%'
+                            }
+                        }
+                    }]
+                }, {
+                    cells: [{
+                        id: 'network-opt',
+                        height: 300,
+                        responsive: {
+                            small: {
+                                width: '100%'
+                            },
+                            medium: {
+                                width: '50%'
+                            },
+                            large: {
+                                width: '50%'
+                            }
+                        }
+                    }, {
+                        id: 'disk-opt',
+                        height: 300,
+                        responsive: {
+                            small: {
+                                width: '100%'
+                            },
+                            medium: {
+                                width: '50%'
+                            },
+                            large: {
+                                width: '50%'
+                            }
+                        }
+                    }]
+                }]
+            }]
+        },
+        components: [{
+            cell: 'instance',
+            type: 'HTML',
+            title: 'Instance type:',
+            elements: [{
+                tagName: 'span'
+            }, {
+                tagName: 'p',
+                textContent: instance.InstanceType
+            }]
+        }, {
+            cell: 'zone',
+            type: 'HTML',
+            title: 'Zone:',
+            elements: [{
+                tagName: 'span'
+            }, {
+                tagName: 'p',
+                textContent: instance.Zone
+            }]
+        }, {
+            cell: 'ami',
+            type: 'HTML',
+            title: 'AMI:',
+            elements: [{
+                tagName: 'span'
+            }, {
+                tagName: 'p',
+                textContent: instance.AMI
+            }]
+        }, {
+            cell: 'os',
+            type: 'HTML',
+            title: 'OS:',
+            elements: [{
+                tagName: 'span'
+            }, {
+                tagName: 'p',
+                textContent: instance.OS
+            }]
+        }, {
+            cell: 'disk-usage',
+            title: 'Disk usage',
+            type: 'Highcharts',
+            connector: {
+                id: 'instanceDetails'
+            },
+            columnAssignment: {
+                index: 'x',
+                MediaGB: 'y',
+                RootGB: 'y',
+                Documents: 'y',
+                Downloads: 'y'
+            },
```

Comment:
Like [Stian commented](https://github.com/highcharts/highcharts/pull/20132#pullrequestreview-1831791062), this chart would be better with labels instead of a legend. Color matching alone is not a11y compliant - it's hard for the vision impaired, and adds another layer of cognitive load for the rest of us. We could use a single series with categories instead: https://jsfiddle.net/highcharts/y0rbwdpa/

PS: We could also consider including data labels, that would further improve accessibility.